examples/tests/doublefree.py

```diff
@@ -19,6 +20,8 @@ class DoubleFreeTest(Test):
 
     :param source: name of the source file located in deps path
     """
+    GLIBC_MSG = 'double free or corruption'
+    MAC_LIBC_MSG = 'pointer being freed was not allocated'
```

Comment:
I understand the concern, but with 2 variables, it's hardly a pressing one. I'll change it.